Swiftfin tvOS/Views/VideoPlayer/Overlays/Overlay.swift

```diff
@@ -33,17 +33,17 @@ extension VideoPlayer {
         var body: some View {
             ZStack {
 
-                MainOverlay()
-                    .visible(currentOverlayType == .main)
-
-                ConfirmCloseOverlay()
-                    .visible(currentOverlayType == .confirmClose)
-
-                SmallMenuOverlay()
-                    .visible(currentOverlayType == .smallMenu)
-
-                ChapterOverlay()
-                    .visible(currentOverlayType == .chapters)
+//                MainOverlay()
+//                    .visible(currentOverlayType == .main)
+//
+//                ConfirmCloseOverlay()
+//                    .visible(currentOverlayType == .confirmClose)
+//
+//                SmallMenuOverlay()
+//                    .visible(currentOverlayType == .smallMenu)
+//
+//                ChapterOverlay()
+//                    .visible(currentOverlayType == .chapters)
```

Comment:
3. Temporarily commented out some Overlay views. 

If this code exists, I run into the following crash, on BottomBarView:

```
Thread 1: Fatal error: No ObservableObject of type VideoPlayerViewModel found. A View.environmentObject(_:) for VideoPlayerViewModel may be missing as an ancestor of this view.
```